docs/reference/ml/ml-shared.asciidoc

```diff
@@ -483,6 +483,11 @@ Identifier for the {dfeed}. It can be a {dfeed} identifier or a wildcard
 expression.
 end::datafeed-id-wildcard[]
 
+tag::decompress-definition[]
+Specifies if the included model definition should be returned as a JSON map or 
```

Comment:
I found it took me a re-read to figure out what "true" mapped to, so it might be helpful to be more explicit:
```suggestion
Specifies whether the included model definition should be returned as a JSON map (`true`) or 
```